packages/gatsby/src/commands/develop.ts

```diff
@@ -288,7 +288,7 @@ module.exports = async (program: IProgram): Promise<void> => {
       const data = await getService(program.directory, `developproxy`)
       const port = data?.port || 8000
       console.error(
-        `Looks like develop for this site is already running. Try visiting http://localhost:${port}/ maybe?`
+        `Looks like develop for this site is already running, can you visit http://localhost:${port}? If it is not, try again in five seconds!`
```

Comment:
I would suggest to put a space between the url and the `?`, as otherwise people might be visiting `localhost:5000?`, since that's a valid url. Depends on the IDE/parser, of course.